.idea/.gitignore

```diff
@@ -0,0 +1,2 @@
+# Default ignored files
```

Comment:
Hey, if you could remove everything under .idea.

Reply:
you can do it manually here (6 commits) and then we squash it... but it would be better if you do it in your terminal